python/lsst/jointcal/jointcal.py

```diff
@@ -25,8 +24,11 @@
 
 __all__ = ["JointcalConfig", "JointcalTask"]
 
+Photometry = collections.namedtuple('Photometry', ('fit', 'model'))
+Astrometry = collections.namedtuple('Astrometry', ('fit', 'model', 'sky_to_tan_projection'))
 
-class JointcalRunner(pipeBase.TaskRunner):
+
+class JointcalRunner(pipeBase.ButlerInitializedTaskRunner):
```

Comment:
The docstring for this is also copied verbatim from HSC MosaicRunner, so it's making reference to things here that don't exist (MosaicTask) so I think this is confusing.